src/components/Input/readme.md

```diff
@@ -17,6 +17,33 @@ const containerStyles = {
 />;
 ```
 
+##### input base with a value controlled through a state and an initial value set by default
+
+```js
+import React, { useState } from 'react';
+import { Input } from 'react-rainbow-components';
+
+const [ value, setValue ] = useState('John Doe');
+
+const handleChange = (e) => {
+    setValue(e.target.value)
+}
+
+const containerStyles = {
+    maxWidth: 700,
+};
+
+<Input
+    id="input-component-1"
```

Comment:
The id must be unique on a web page, the first example already use `id="input-component-1"`
Also keep in mind when adding new examples and use the ids that the integration tests depend on this ids and on the example order too